app/assets/stylesheets/bastion/tables.scss

```diff
@@ -20,8 +20,8 @@
   
   th.row-select,
   td.row-select {
-    width: 30px;
-    max-width: 30px;
+    width: $row-select-width;
```

Comment:
I guess `variables.scss` needs to be imported?  @ehelms do you have any ideas about why the tests are failing?